platform/app/about/layout.tsx

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
Layout page is not necessary. Layout is used for general themes and this route ( / )  has one. Could u delete this?